Pulumi.github.yaml

```diff
@@ -5,3 +5,5 @@ config:
     secure: AAABAJoOP517E0wIlRNaJM90ozYbdTffr1Nfqnz3cSnD2kAWck1akvfgFazvM5qO7LocIAM2dmyaBxbaBK9zpiNmSOpDA+4m
   holochain:hra2PulumiAccessToken:
     secure: AAABAESpY2fKLjLgiRFaQqZ1pE+H+SFfYbkKo2jKidYtZQT5KK26u3VKW/jmlG6JQTHebMt+M1OSgYk/Bt8r8UDjKbma6gbcwr/K6w==
+  github:token:
```

Comment:
What's this for? I don't see it being used?

Reply:
Sorry removed, wasn't meant to commit this, was trying out different cli commands